### DISCUSSION TITLE
691 paths removed from GEDCOM 5.5.1  - limited backward compatability.

### DISCUSSION THREAD

**Discussion by augean** [FamilySearch/GEDCOM#428] [Q&A] [answered]
stillthere = 2306
removed =691
Almost a  1/4 of  GEDCOM 5  paths are removed in GEDCOM 7

Attached is a list of paths that have been removed,  and paths that are still there.

[stillthere.txt](https://github.com/FamilySearch/GEDCOM/files/13999527/stillthere.txt)
[removed.txt](https://github.com/FamilySearch/GEDCOM/files/13999528/removed.txt)

looks like there is  limited backward compatibility
many GEDCOM 5.5.1 files can't be used in the GEDCOM7 schema.

the conversion would be  complex from GEDCOM 5 to GEDCOM7.
Any suggestions  for conversion of GEDCOM 5 to GEDCOM 7 files?

**Comment by tychonievich** [comment, marked as answer]
You are correct; 7.0 is a major (not minor or patch) version, and hence not backwards compatible with earlier versions. A few of the removed paths are removed entirely (e.g. SUBN), but most are refactored into another structure (e.g. FONE → TRAN). We have a guide to conversion from 5.5.1 at https://gedcom.io/migrate/ and a few community-supplied example converters at https://gedcom.io/tools/#v551-to-v70-conversion-code

FYI, saying "GEDCOM 5" is technically vague; 5.5.1, 5.5, 5.4, 5.3, and 5.0 were not backwards compatible either. We do not currently have a guide for conversion from 5.5 or earlier.

**Comment by augean** [comment]
its   actually    Release 5.5.1      which     I used for the comparison
